lib/parser.js

```diff
@@ -143,6 +143,15 @@ function parseFromUrl(url, fetchOptions, options) {
   });
 }
 
+/**
+ * 
+ * @param {object} refParser refParser 
+ * @param {object} parsedJSON parsed AsyncAPI document
+ * @param {string} initialFormat information of the document was originally JSON or YAML
+ * @param {string} asyncapiYAMLorJSON An AsyncAPI document in JSON or YAML format.
```

Comment:
Same comment in [here](https://github.com/asyncapi/parser-js/pull/305/files#r701450076), what do you think @arjungarg07 ?